tests/tt_eager/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+
+set(TT_EAGER_TESTS_DTX
+    dtx/collapse_transformations
+    dtx/overlap
+    dtx/tensor
+    dtx/test_dtx_tilized_row_to_col_major
+    dtx/test_dtx
+    dtx/unit_tests
+)
+# tensor_evaluate                    # <- not called in run_tt_eager.py
+
+set(TT_EAGER_TESTS_OPS
+    # ccl/test_all_gather_utils           # <- not called in run_tt_eager.py
+    ops/test_average_pool
+    ops/test_eltwise_binary_op
+    ops/test_eltwise_unary_op
+    ops/test_softmax_op
+    ops/test_layernorm_op
+    # ops/test_moreh_adam_op                  # <- not called in run_tt_eager.py
```

Comment:
https://github.com/tenstorrent-metal/tt-metal/issues/7272
https://github.com/tenstorrent-metal/tt-metal/issues/7274

I don't see `test_all_gather_utils` run anywhere but it might just be run locally